src/routers/swap_quote_router.ts

```diff
@@ -0,0 +1,12 @@
+import { Orderbook, SupportedProvider } from '@0x/asset-swapper';
+import * as express from 'express';
+import * as asyncHandler from 'express-async-handler';
+
+import { SwapQuoteHandlers } from '../handlers/swap_quote_handlers';
+
+export const createSwapQuoteRouter = (orderBook: Orderbook, provider: SupportedProvider): express.Router => {
```

Comment:
Same comment as in `SwapQuoteHttpService` maybe this should be named `createSwapRouter`